Doc entries for Http Proxy authentication (#23316)
trinodb/trino: docs/src/main/sphinx/admin/properties-http-client.md

Host and port for an HTTP proxy with the format `example.net:8080`.

### `http-client.http-proxy.user`

- **Type:** {ref}`prop-type-string`

> Username for HTTP proxy (Basic authentication type only supported).

Reviewer: ```suggestion
Username for basic authentication with the HTTP proxy.
```

### `http-client.http-proxy.password`

- **Type:** {ref}`prop-type-string`

> Password for HTTP proxy (Basic authentication type only supported).

Reviewer: ```suggestion
Password for basic authentication with HTTP proxy.
```
